src/azure-cli/azure/cli/command_modules/acs/_params.py

```diff
@@ -202,6 +203,7 @@ def load_arguments(self, _):
                    validator=validate_load_balancer_idle_timeout)
         c.argument('outbound_type', arg_type=get_enum_type([CONST_OUTBOUND_TYPE_LOAD_BALANCER,
                                                             CONST_OUTBOUND_TYPE_USER_DEFINED_ROUTING]))
+        c.argument('auto_upgrade_channel', arg_type=get_enum_type([CONST_RAPID_UPGRADE_CHANNEL, CONST_STABLE_UPGRADE_CHANNEL, CONST_PATCH_UPGRADE_CHANNEL, CONST_NODE_IMAGE_UPGRADE_CHANNEL, CONST_NONE_UPGRADE_CHANNEL]))
```

Comment:
Do we need to declare the default value here? Such as 
```
arg_type=get_enum_type(...),..., default='xxx'
```

Reply:
@xhl873, assuming the answer is no. There was no default before.

Reply:
Can we use upgrade channel none as the default?

Reply:
So, nil is used for unspecified, and None is used if the customer wants to turn it off.

We could make None the default for create, since there is nothing it would be turning off, but for update adding None as the default would make any update call turn off autoupgrade for the customer unless they specified it on each call. 

I talked with Xiahe about this as we agreed that just adding a default like this for create did not make sense.

Reply:
The updated model includes an enum value for unspecified

Reply:
I could see making unspecified the default. Although, rejecting nil in place of a new unspecified value at this point would still break the preview CLI, and current behavior of the REST API if customers were using it directly. (ok, because autoupgrader is not GA yet?)

Reply:
Also, using unspecified over nil adds another channel to track where nil could be used, but I get the argument around removing the reliance on nil values.

Reply:
Not sure it matters too much, we can convert the nil to unspecified in RP ( or keep using nil )